src/MongoDB.Driver.Core/ReadPreferenceHedge.cs

```diff
@@ -61,9 +61,9 @@ public ReadPreferenceHedge(bool isEnabled)
 
         // public methods
         /// <inheritdoc/>
-        public override bool Equals(object other)
+        public override bool Equals(object obj)
```

Comment:
This one is interesting.

Apparently when you `<inheritdoc/>` you have to use the same param name as the base class method.